consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/Gossiper.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.consensus.ibft;
+
+import tech.pegasys.pantheon.ethereum.core.Address;
+import tech.pegasys.pantheon.ethereum.p2p.api.Message;
+import tech.pegasys.pantheon.ethereum.p2p.api.MessageData;
+
+import java.util.List;
+
+public interface Gossiper {
+
+  boolean gossipMessage(Message message);
+
+  boolean send(MessageData messageData, List<Address> excludeAddressesList);
```

Comment:
This form of gossip where we haven't got the original message is never going to have excluded addresses so perhaps just remove that argument?